src/functionalTest/resources/scenarios/RIA-3006-EA-appeal-submission-fee.json

```diff
@@ -0,0 +1,32 @@
+{
+  "description": "RIA-3006 Should display the fee amount for EA appeal type",
+  "request": {
+    "uri": "/asylum/ccdAboutToStart",
+    "credentials": "LegalRepresentative",
+    "input": {
+      "eventId": "submitAppeal",
+      "state": "appealStarted",
+      "caseData": {
+        "template": "minimal-appeal-started.json",
+        "replacements": {
+          "appealType": "refusalOfEu",
+          "homeOfficeDecisionDate": "{$TODAY-14}"
+        }
+      }
+    }
+  },
+  "expectation": {
+    "status": 200,
+    "errors": [],
+    "caseData": {
+      "template": "minimal-appeal-started.json",
+      "replacements": {
+        "submissionOutOfTime": "No",
+        "appealType": "refusalOfEu",
+        "homeOfficeDecisionDate": "{$TODAY-14}"
+      },
+      "appealFeeDesc": "The fee for this type of appeal with a hearing is £140.00",
```

Comment:
Is this always going to be the same?  E.g.  in AAT will this value change and make the test fail intermittently?